ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/affordances/RDXArmManager.java

```diff
@@ -171,6 +163,14 @@ public void update(boolean interactablesEnabled)
                   desiredHandPoseChanged |= armIKSolvers.get(side).getDesiredHandControlPoseChanged();
                }
             }
```

Comment:
fixed bug, if the wrench calculator is null, it will skip the armIKSolver part. This means that the IK for the selectable hand didn't work if the robot has no hand